### DISCUSSION TITLE
Great Idea.

### DISCUSSION THREAD

**Discussion by chrisstpierre** [OpenSignLabs/OpenSign#136] [General]
I think this is a good idea. This market has too many competitive each charging too much. self-hosting is ideal for those that can manage. Could maybe be a nonprofit ?

**Comment by andrew-opensignlabs** [comment]
You are right. Non-profits can use this solution. We are getting to know a lot of new use cases of our solution as people are reaching out to us with their own ideas of implementing it in their situations.
